pkg/context/context.go

```diff
@@ -21,14 +21,17 @@ const (
 type CommandBuilder func(name string, arg ...string) *exec.Cmd
 
 type Context struct {
-	Debug        bool
-	DebugPlugins bool
-	verbose      bool
-	FileSystem   *afero.Afero
-	In           io.Reader
-	Out          io.Writer
-	Err          io.Writer
-	NewCommand   CommandBuilder
+	Debug                  bool
+	DebugPlugins           bool
+	RunPlugInInDebugger    string
```

Comment:
What do we think about creating a new struct to house plugin debugging config (e.g. all of the new values added here)?  Then we could add the one `DebugConfig` (or similar)  struct to `Context` here, keeping down the number of member fields.

Reply:
Done !!